Woof/Woof/ViewModel/LoginViewModel.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// An interface that provides interaction between the `LoginView` and the model layer.
```

Comment:
What? :)

Reply:
Please read what is the view model and explain in the doc comments what the `LoginViewModel` is for.

Reply:
read about viewmodel and changed the documentation